e2e/go.mod

```diff
@@ -248,3 +248,5 @@ replace github.com/cosmos/ibc-go/v7 => ../
 replace github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+
+replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.47.5-0.20230828070859-c9144f02dda8
```

Comment:
how come this is being changed?

Reply:
0.47.4 doesn't have `proposal.SetMsgs` needed here.